### ISSUE TITLE
Google backend will not work forever

### ISSUE THREAD

**Issue by tusing** [tusing/reddit-ffn-bot#107]
When the bot receives a request, it searches for that request in Google (adding on, for example, "site:fanfiction.net" to the query if the request was for fanfiction.net).

This script will not work forever. Google doesn't really have an API and it's cracking down on script-based access. To ensure the stability of the bot, we'll want to use a sanctioned API-based method of crawling Google.

An option might be Google's [Custom Search](https://developers.google.com/custom-search/docs/overview) APIs, where we define Custom Search over the two or three websites we support. We would then replace our call to the Google script with a call to Custom Search.

**Comment by ThrawnCA** [comment]
Are there alternative search engines that could do the job? StartPage, DuckDuckGo, etc?

**Comment by Thatgirl318** [comment]
Is there any reason we can't use the fanfiction search funciton? Is it just bad or lack of API also?

**Comment by Thatgirl318** [comment]
We might also be able to use the Custom Search Restriction json API

**Comment by tusing** [comment]
The reason is that I have no time to work on this anymore :( But I will gladly take pull requests that fix the search functionality!

**Comment by Thatgirl318** [comment]
Yeah totally (once finals are done). I just didn't want to go down a path you tried and already abandoned.

**Comment by Thatgirl318** [comment]
Is it possible for me to get the logs?
